wallet/udb/tx_test.go

```diff
@@ -120,6 +120,7 @@ func TestInsertsCreditsDebitsRollbacks(t *testing.T) {
 	// the double spend.
 	spendingTx := wire.NewMsgTx()
 	spendingTxIn := wire.NewTxIn(wire.NewOutPoint(TstDoubleSpendTx.Sha(), 0, dcrutil.TxTreeRegular), []byte{0, 1, 2, 3, 4})
+	spendingTxIn.ValueIn = TstDoubleSpendTx.MsgTx().TxOut[0].Value
```

Comment:
I realize this file is `//+build ignore`'d but if we update it we should at least use the new NewTxIn signature